helm/chart/maesh/templates/mesh/mesh-daemonset.yaml

```diff
@@ -57,7 +58,7 @@ spec:
           imagePullPolicy: {{ .Values.tracing.jaeger.image.pullPolicy | default "IfNotPresent" }}
           args:
             - "service"
-            - "-lapp.kubernetes.io/name=jaeger,app.kubernetes.io/component=agent"
```

Comment:
I think we should keep these selectors because these labels should be available on `jaeger` resources deployed by a user. Btw, these labels should not be removed from the `jaeger` resources.